.travis.yml

```diff
@@ -45,8 +46,10 @@ install:
 before_script:
   - git config --global user.email "travis-ci@codeclimate.com"
   - git config --global user.name "Travis CI"
+  - mkdir -p "$HOME/.php-cs-fixer"
 
 script:
+  - if [[ "$WITH_CS" == "true" ]]; then vendor/bin/php-cs-fixer fix --config=.php_cs --verbose --diff --dry-run; fi
```

Comment:
It should suffice to run it once, on the minimum version supported by us, right?